scripts/FilterIconsCatalog.py

```diff
@@ -0,0 +1,185 @@
+import os
+import shutil
+import zipfile
+
+set = {
```

Comment:
The filter list of icons we (one consumer) happen to care about probably shouldn't happen in the repo itself. We should discuss offline if it would be better to clone and add filtering in our clone